dlk/python/dlk/core/operators.py

```diff
@@ -1321,6 +1290,17 @@ def run(self, **kwargs) -> Dict:
         kwargs['data'] = scale * x_norm + beta
         return kwargs
 
+    def de_run(self, **kwargs) -> Dict:
+        scale = np.float64(self._input_ops['scale'].data)
+        beta = np.float64(self._input_ops['B'].data)
+        mean = np.float64(self._input_ops['mean'].data)
+        var = np.float64(self._input_ops['var'].data)
+
+        kwargs['nega_idx'] = [v for v in range(len(scale)) if scale[v] < 0]
```

Comment:
Since it was more straightforward to retrieve the values from the batch norm rather than the threshold skipping pass, but I agree this will go to the optimization pass.

Reply:
Now the `nega_idx` is moved inside the optimization pass with https://github.com/blue-oil/blueoil/pull/53/commits/a7f73bc522a476b4630a7d72e243152e6a0d59e0